source/systems/research.md

```diff
@@ -93,18 +104,19 @@ You can only choose one column 6 research in each of the Combat, Civilian, and T
 | Provost | Duelist | Combat Academy level 5; 16 diamonds | Increases Knights' armor by 50% |
 | Improved Evasion | Evasion | Archery level 5; 16 diamonds | Increases Archers' armor by 50% |
 | Steel Armor | Iron Armor | Town Hall level 5; 64 iron ingots | Increases Guards' armor's durability by 50% |
+| Plate Armor | Iron Armor | Blacksmith level 4; 32 iron ingots | Blacksmith Learns Plate Armor Recipes |
```

Comment:
No title case for the wiki descriptions
`Blacksmith learns [plate armor](../../source/items/platearmor) recipes` prolly